backend/app.js

```diff
@@ -12,6 +13,11 @@ let tagsRouter = require('./routes/tags');
 
 let app = express();
 
+// For Passport
+app.use(session({ secret: 'keyboard cat',resave: true, saveUninitialized:true})); // session secret
```

Comment:
so this is only useful if we decide to use sessions. I removed the functionality from the actual logic functions since it's not necessary right now. I can comment out this line if you want